site/content/en/docs/Guides/Client SDKs/_index.md

```diff
@@ -316,6 +316,103 @@ or SDK.WatchGameServer() instead to list the connected players.
 
 [playerstatus]: {{< ref "/docs/Reference/agones_crd_api_reference.html#agones.dev/v1.PlayerStatus" >}}
 
+### Counters And Lists
+
+{{< alpha title="Counters And Lists" gate="CountersAndLists" >}}
+
+The `Counters` and `Lists` features in the SDK offer a flexible configuration for tracking various entities like 
+players, rooms, and sessions. However, it's important to note that this feature has been in Alpha stage and disabled
+by default.
+
+{{< alert title="Note" color="info">}}
+There is a potential race condition when count values are set from both the SDK and through the K8s API(Allocation or otherwise),
+since the SDK update operation back to the CRD value is batched asynchronous any value incremented past the capacity will be
+silently truncated to capacity.
+{{< /alert >}}
+
+### Counters
+
+#### Alpha().GetCounterCount(key)
+
+This function retrieves the current count for a specified Counter in the game server. This returns `-1` an `error` if 
+the key is not predefined.
+
+{{< alert title="Note" color="info">}}
```

Comment:
We can move this note up to the top under `Counters And Lists` and make the language more generic, since this is the case for any of the "get" operations for Counters and Lists.

Reply:
I've modified the content to make it more generic and relocated it under the title.👍🏻